src/coreclr/debug/daccess/task.cpp

```diff
@@ -5243,7 +5243,7 @@ EnumMethodInstances::CdStart(MethodDesc* methodDesc,
                              CLRDATA_ENUM* handle)
 {
     if (!methodDesc->HasClassOrMethodInstantiation() &&
-        !methodDesc->HasNativeCodeReJITAware())
+        !(g_pEEInterface->GetFunctionAddress(methodDesc) != NULL))
```

Comment:
Right now this is testing if the method has a default or active version with code. I suspect the best behavior is to test if there is any NativeCodeVersion that contains code, regardless whether it is default or active.